repository.rdf

```diff
@@ -397,6 +397,17 @@
 
 
 
+    <!-- http://fedora.info/definitions/v4/repository#RepositoryRoot -->
+
+    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#RepositoryRoot">
+        <rdfs:label xml:lang="en">repository root</rdfs:label>
+        <owl:subClassOf rdf:resource="http://fedora.info/definitions/v4/repository#Thing"/>
+        <owl:disjointWith rdf:resource="http://fedora.info/definitions/v4/repository#Resource"/>
```

Comment:
Yeah, I think it goes the other way-- to my understanding, a root is a `Resource`. I emphasize _a_ root, not _the_ root, because the way the discussions for the CRUD piece of the specs are going, we will contemplate multiply-rooted repos.